csharp/ql/test/experimental/ir/ir/raw_ir.expected

```diff
@@ -1327,8 +1327,8 @@ pointers.cs:
 #    6|     r6_1(glval<Int32*>)  = VariableAddress[b]          : 
 #    6|     r6_2(glval<Int32[]>) = VariableAddress[arr]        : 
 #    6|     r6_3(Int32[])        = Load                        : &:r6_2, ~m?
-#    6|     r6_4(Int32*)         = Convert                     : r6_3
-#    6|     mu6_5(Int32*)        = Store                       : &:r6_1, r6_3
+#    6|     r6_4(Int32*)         = CheckedConvertOrThrow       : r6_3
```

Comment:
I'm unsure what this change actually means. Or if it is expected at all.

Reply:
I think it is fine; the IR would [insert a `Convert` operation](https://github.com/github/codeql/blob/main/csharp/ql/src/experimental/ir/implementation/raw/internal/TranslatedInitialization.qll#L142) when the RHS of an assignment did not match the LHS, but this is no longer needed. However, now there is an implicit cast, which translates to the `CheckedConvertOrThrow` operation.